client/dependencies.go

```diff
@@ -86,3 +42,49 @@ type DependencyList struct {
 
 	Items []Dependency `json:"items" protobuf:"bytes,2,rep,name=items"`
 }
+
+type DependenciesInterface interface {
+	List(opts api.ListOptions) (*DependencyList, error)
+}
+
+type dependencies struct {
+	rc *restclient.RESTClient
+}
+
+func newDependencies(c restclient.Config) (*dependencies, error) {
+	c.APIPath = "/apis"
```

Comment:
I am not sure about this bit, maybe we should provide helper function which populates the config object and returns it? We could get rid of redundant code in client/resdef.go:57 too (especially after k8s1/k8s2 bit is not needed anymore.).